master/internal/api_runs.go

```diff
@@ -382,41 +388,106 @@ func (a *apiServer) MoveRuns(
 		if err != nil {
 			return nil, err
 		}
+		tx, err := db.Bun().BeginTx(ctx, nil)
+		if err != nil {
+			return nil, err
+		}
+		defer func() {
+			txErr := tx.Rollback()
+			if txErr != nil && txErr != sql.ErrTxDone {
+				log.WithError(txErr).Error("error rolling back transaction in MoveExperiments")
```

Comment:
nit (non-blocking): should this include something about while attempting to move runs for clarity?

Reply:
Updated